config/packages/assets.yaml

```diff
@@ -1,4 +1,4 @@
 framework:
     assets:
-        json_manifest_path: '%kernel.project_dir%/public/build/manifest.json'
+        base_path: '/'
         strict_mode: '%kernel.debug%'
```

Comment:
strict_mode is now dead code, as it is used only for the JsonManifestStrategy

Reply:
OK. I've removed it. Thanks.

However, I don't understand why `json_manifest_path` worked before but it's failing now.

Reply:
it is still working now in non-strict mode, by being a passthrough.

Before, there was no strict mode.